resources/qml/Menus/ConfigurationMenu/CustomConfiguration.qml

```diff
@@ -345,8 +346,8 @@ Item
                         anchors.left: warningImage.right
                         anchors.leftMargin: UM.Theme.getSize("default_margin").width
                         width: selectors.controlWidth - warningImage.width - UM.Theme.getSize("default_margin").width
-                        text: catalog.i18nc("@label", "Use glue for better adhesion with this material combination.")
-                        visible: CuraSDKVersion == "dev" ? false : warnings.buildplateCompatibilityError || warnings.buildplateCompatibilityWarning
+                        text:warnings.coreCompatibilityWarning? catalog.i18nc("@label", "Combination not recommended. Load BB core to slot 1 (left) for better reliability."): catalog.i18nc("@label", "Use glue for better adhesion with this material combination.")
+                        visible: CuraSDKVersion == "dev" ? false : warnings.buildplateCompatibilityError || warnings.buildplateCompatibilityWarning|| warnings.coreCompatibilityWarning
                         wrapMode: Text.WordWrap
                     }
```

Comment:
Why not add a different UM.Label here with a different visibility check

Something like this i mean
```python
                    UM.Label
                    {
                        id: materialCompatibilityLabel
                        anchors.left: warningImage.right
                        anchors.leftMargin: UM.Theme.getSize("default_margin").width
                        width: selectors.controlWidth - warningImage.width - UM.Theme.getSize("default_margin").width
                        text: catalog.i18nc("@label", "Use glue for better adhesion with this material combination.")
                        visible: CuraSDKVersion == "dev" ? false : warnings.buildplateCompatibilityError || warnings.buildplateCompatibilityWarning
                        wrapMode: Text.WordWrap
                    }
                    
                    UM.Label
                    {
                        id: materialCompatibilityLabel
                        anchors.left: warningImage.right
                        anchors.leftMargin: UM.Theme.getSize("default_margin").width
                        width: selectors.controlWidth - warningImage.width - UM.Theme.getSize("default_margin").width
                        text: catalog.i18nc("@label", "Combination not recommended. Load BB core to slot 1 (left) for better reliability.")
                        visible: warnings.coreCompatibilityWarning
                        wrapMode: Text.WordWrap
                    }
```

Reply:
done 6542d9ff1334d176d75ed9b2a8e7aba633f88040